Service/ResponseTransformerStrategy/ExpectedResponseStrategy.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service\ResponseTransformerStrategy;
+
+use Auto1\ServiceAPIClientBundle\Service\Deserializer;
+use Auto1\ServiceAPIClientBundle\Service\ResponseTransformerStrategy;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+
+class ExpectedResponseStrategy implements ResponseTransformerStrategy, LoggerAwareInterface
```

Comment:
there is no use to add logging capabilities to this class, better to remove the LoggerAwareInterface

Reply:
removed